src/Pages/Articles.jsx

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect, useState } from 'react'
+import { fetchAllArticles } from '../../api/requests'
+import ArticleCard from '../components/ArticleCard'
+import Header from '../components/Header'
+import Navbar from '../components/Navbar'
+import Sticky from '../components/Sticky'
+
+const Articles = () => {
+    const [articles,setArticles] = useState([])
+
```

Comment:
Please add a state for loading and display a message to the user while the articles are loading.